lib/resources/example_json/r4_examples_document_reference.rb

```diff
@@ -140,148 +154,170 @@ module Resources
             }
           ],
           'text': 'Clinical Note'
+        },
+        {
+          'coding': [
+            {
+              'system': 'http://loinc.org',
+              'code': '83805-2'
+            }
+          ]
         }
       ],
       'subject': {
-        'reference': 'Patient/12742400',
-        'display': 'PETERS, TIM A'
+        'reference': 'Patient/2798003',
+        'display': 'Parnell, Stephen M'
       },
       'author': [
         {
-          'reference': 'Practitioner/12742069',
-          'display': 'Portal, Portal'
+          'reference': 'Practitioner/2797943',
+          'display': 'Braus, Sasha'
         }
       ],
       'authenticator': {
-        'reference': 'Practitioner/12742069',
-        'display': 'Portal, Portal'
+        'reference': 'Practitioner/2797943',
+        'display': 'Braus, Sasha'
       },
       'content': [
         {
           'attachment': {
             'contentType': 'application/pdf',
-            'url': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/XR-197411797',
-            'title': 'Auto TC Summary',
-            'creation': '2020-12-10T07:19:54.000Z'
+            'url': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/XR-21961261',
+            'title': 'Admission H & P',
+            'creation': '2021-03-12T22:35:03.000Z'
+          }
+        },
+        {
+          'attachment': {
+            'contentType': 'text/html',
+            'url': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/TR-21961261',
+            'title': 'Admission H & P',
+            'creation': '2021-03-12T22:35:03.000Z'
           }
```

Comment:
We currently do not support sending out a TR attachment on patient access. Can we remove this please. Thanks!

Reply:
Updated here: be2e86f8df5e2a32e76222796e550f5a86c17500